dockstore-cli-integration-testing/src/test/java/io/dockstore/client/cli/YamlClientIT.java

```diff
@@ -0,0 +1,71 @@
+package io.dockstore.client.cli;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import com.google.common.collect.Lists;
+import io.dockstore.common.FlushingSystemErrRule;
+import io.dockstore.common.FlushingSystemOutRule;
+import io.dockstore.common.TestUtility;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.contrib.java.lang.system.ExpectedSystemExit;
+import org.junit.contrib.java.lang.system.SystemErrRule;
+import org.junit.contrib.java.lang.system.SystemOutRule;
+
+import static org.junit.Assert.assertTrue;
+
+public class YamlClientIT {
+
+    @Rule
+    public final SystemOutRule systemOutRule = new FlushingSystemOutRule().enableLog().muteForSuccessfulTests();
+
+    @Rule
+    public final SystemErrRule systemErrRule = new FlushingSystemErrRule().enableLog().muteForSuccessfulTests();
+
+    @Rule
+    public final ExpectedSystemExit systemExit = ExpectedSystemExit.none();
+
+    @Test
+    public void testAllYamlHelpMessages() throws IOException {
+        checkCommandForHelp(new String[] { "yaml"});
+        checkCommandForHelp(new String[] { "yaml", "--help" });
+        checkCommandForHelp(new String[] { "yaml", "validate", "--help" });
+    }
+
+    private void checkCommandForHelp(String[] argv) throws IOException {
+        final ArrayList<String> strings = Lists.newArrayList(argv);
+        strings.add("--config");
+        strings.add(TestUtility.getConfigFileLocation(true));
+
+        Client.main(strings.toArray(new String[0]));
+        System.out.println(strings.toArray(new String[0]));
+        Assert.assertTrue(systemOutRule.getLog().contains("Usage: dockstore"));
+        systemOutRule.clearLog();
+    }
+
+    @Test
+    public void missingPathParamter() {
+        Client.main(new String[] { "yaml", "validate" });
+        Assert.assertTrue(systemOutRule.getLog().contains("ERROR: Missing --path <path>"));
+        systemOutRule.clearLog();
+    }
+
+    @Test
+    public void completeRun() {
+        Client.main(new String[] { "yaml", "validate", "--path", "../dockstore-client/src/test/resources/YamlVerifyTestDirectory/correct-directory" });
+        assertTrue(systemOutRule.getLog().contains("src/test/resources/YamlVerifyTestDirectory/correct-directory/.dockstore.yml is a valid yaml file"));
```

Comment:
these should use a string constant imported from your non-test code, that way a developer can quickly tell what test to modify when changing the return message